packages/kosu-wrapper-enhancements/README.md

```diff
@@ -0,0 +1,83 @@
+# Wrapper Enhancemnts
+
+A TypeScript/JavaScript library for assistance with interactions with the Kosu contract system wrappers. This library simplifies the
```

Comment:
Unfinished sentence?

I will probably approve anyway since this is such a large refactor, and you can fix in a separate PR.

Reply:
This may have been me?

Reply:
This was me.  I either got distracted or hit undo on the wrong screen.  I have pushed a fix in #358